hunts/src/puzzlesSlice.js

```diff
@@ -96,6 +96,30 @@ function puzzleComparator(a, b) {
   } else if (a.is_meta && !b.is_meta) {
     return 1;
   }
+  // High-priority first
```

Comment:
If it were me, I would:

1) Write a helper function `priority(row)` that returns 1 if hi-pri, else if-1 if lo-pri, else 0.
2) Return `priority(a) - priority(b)` if nonzero. Else, fall back to b.id - a.id;

Reply:
Latest commit follows your suggestion.